app/recordtransfer/templates/recordtransfer/email/email_base.html

```diff
@@ -0,0 +1,104 @@
+{% load i18n %}
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8">
+        <meta name="description"
+              content="Email communication from the National Centre for Truth and Reconciliation.">
+        <title>{% blocktrans %}Email from Our Company{% endblocktrans %}</title>
+    </head>
+    <body style="margin: 0;
+                 padding: 0;
+                 background-color: #f4f4f4;
+                 font-family: 'Merriweather', Georgia, 'Times New Roman', serif">
+        <div style="background-color: #ffffff;
+                    margin: 40px auto;
+                    padding: 30px;
+                    width: 600px">
+            <!-- Logo -->
+            <div style="text-align: center;">
+                <img src="https://ehprnh2mwo3.exactdn.com/wp-content/uploads/2023/08/NCTR-BILINGUAL-sh.png"
+                     alt="{% blocktrans %}Company Logo{% endblocktrans %}"
+                     style="width: 80%;
+                            height: auto">
+            </div>
+            <!-- Header Image -->
+            <div style="text-align: center; margin-top: 20px;">
+                <img src="https://ehprnh2mwo3.exactdn.com/wp-content/uploads/2021/02/hero4-scaled.jpg"
+                     alt="{% blocktrans %}Company Image{% endblocktrans %}"
+                     style="width: 80%;
+                            height: auto">
+            </div>
+            <div style="text-align: center;
+                        font-weight: bold;
+                        color: #89181A;
+                        font-size: 20pt;
+                        padding: 6px 20px;
+                        background-color: #FFFFFF;
+                        margin-bottom: 15px;
+                        margin-top: 20px;
+                        font-family: 'Merriweather', Georgia, 'Times New Roman', serif">
+                {% block title %}<!-- Your dynamic title here -->{% endblock %}
+            </div>
+            <div style="font-size: 10pt;
+                        font-family: 'Merriweather', Georgia, 'Times New Roman', serif;
+                        padding: 0 20px">
+                {% block content %}<!-- Your dynamic content here -->{% endblock %}
+            </div>
+        </div>
+        <div style="text-align: center;
+                    font-size: 10px;
+                    color:#ccc8c8;
+                    font-family:  serif;
+                    margin-top: 40px;
+                    text-transform: uppercase;
+                    line-height: 0.8">
+            <p>
+                {% blocktrans %}National Centre for Truth and Reconciliation, 177 Dysart Rd, Winnipeg, MB R3T 2M8{% endblocktrans %}
+            </p>
+            <p>
+                {% blocktrans %}Don't like these emails?{% endblocktrans %}
+                <a href="http://{{ site_domain }}{{ profile_url }}"
+                   style="color:#9f9f9f">{% blocktrans %}Unsubscribe{% endblocktrans %}</a>
```

Comment:
Two things here:

1. We don't have a proper unsubscribe functionality, so a person has to log in to their profile to do that. I'd rather the text say something about the user's profile.
2. `{{ profile_url }}` is not defined here, so we need to supply a proper path here.

```suggestion
                {% blocktrans %}Don't like these emails?{% endblocktrans %}
                <a href="http://{{ site_domain }}{% url 'recordtransfer:user_profile' %}"
                   style="color:#9f9f9f">{% blocktrans %}Unsubscribe in your profile{% endblocktrans %}</a>
```